app/Http/Middleware/Localization.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\App;
+
+class Localization
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if (session()->has('locale')) {
+            App::setLocale(session()->get('locale'));
```

Comment:
Qu'est-ce qui se passe si la locale existe pas ? Ça fallback ?

Reply:
cf `fallback_locale` https://laravel.com/docs/5.8/localization#configuring-the-locale

Reply:
https://github.com/fight-food-waste/collects/blob/add-french-translation/config/app.php#L99

Reply:
👍

Reply:
cf 076e0de